packages/uncaught-exception-observer/test/UncaughtExceptionObserver.test.ts

```diff
@@ -61,25 +61,24 @@ async function itObservesUnhandledJavaScriptFetchRejections(): Promise<void> {
         <script src="https://localhost:3000/404.js" async></script>
     `);
 
-	await new Promise((resolve) => setTimeout(resolve, 10));
+	for (let i = 0; i < 10; i++) {
+		await new Promise((resolve) => setTimeout(resolve, 10));
+		if (errorEvent) {
+			break;
+		}
+	}
 
 	observer.disconnect();
 
 	if (!(errorEvent instanceof window.ErrorEvent)) {
 		throw new Error('Error event not dispatched.');
 	}
 
-	if (
-		errorEvent.error.message !==
-		'Fetch to "https://localhost:3000/404.js" failed. Error: connect ECONNREFUSED 127.0.0.1:3000'
-	) {
+	if (!errorEvent.error.message.startsWith('Fetch to "https://localhost:3000/404.js" failed.')) {
```

Comment:
Node.js:
- 16: `Error: connect ECONNREFUSED 127.0.0.1:3000`
- 18: `Error: connect ECONNREFUSED ::1:3000`
- 20: `Error: `
Since the actual message seemed to be irrelevant to this test, I simplified the check to ignore that part.

Reply:
Yes, that should be fine.